assets/js/blocks/cart-checkout/checkout/sidebar/index.js

```diff
@@ -75,7 +76,7 @@ const CheckoutSidebar = ( {
 				currency={ totalsCurrency }
 				values={ cartTotals }
 			/>
-			<ExperimentalOrderMeta.Slot />
+			<ExperimentalOrderMeta.Slot cart={ cart } />
```

Comment:
It'd be nice to avoid having to invoke the hook in this component just so `ExperimentalOrderMeta` can receive the cart as a prop. Could we instead have `ExperimentalOrderMeta` be a wrapper component around the slot and then just expose the data to the slot? Something like:

```js
const ExperimentalOrderMetaSlot = ( props ) => {
  // you could potentially get data from other hooks/context in here to pass along as well?
  return <ExperimentalOrderMeta.Slot cart={ useStoreCart() } { ...props } />
}
```

Reply:
We have a wrapper component `Slot` so we can do that there.